docs/serialization.rst

```diff
@@ -146,7 +146,7 @@ Then, a serializer for it can be implemented as below:
 
 .. code:: python
 
-    from hazelcast.serialization.api import CompactSerializer, CompactWriter, CompactReader
+    from hazelcast.serialization.api import CompactSerializer, CompactWriter, CompactReader, CompactSerializableClass
```

Comment:
Don't forget to change this file if you change `get_class` method's return type.

Reply:
Removed the import for CompactSerializableClass at: https://github.com/hazelcast/hazelcast-python-client/pull/546/commits/ae2532d913aad0f7b02f20cacd0852209ea024c4

Reply:
Looks like you have not removed the import yet.

```suggestion
    from hazelcast.serialization.api import CompactSerializer, CompactWriter, CompactReader
```

Reply:
My comments keep disappearing. Removed that at: https://github.com/hazelcast/hazelcast-python-client/pull/546/commits/df32f6dae22693344c393081f1a7339517051218